tutorials/sigfox-gw/index.md

```diff
@@ -107,17 +106,17 @@ If you can see messages sent by your device, you can proceed to the next step.
 
         $ cd community/tutorials/sigfox-gw
 
-1. Create a new python virtual environment:
+1. Create a new Python 3 virtual environment:
 
-        $ virtualenv venv
+        $ python3 -m venv venv
```

Comment:
Very very minor nit: if you put the venv in `.venv`, instead of `venv`, VS Code (and other editors maybe?) automatically recognizes it

Reply:
venv seems to be the most common standard for python virtual envs.. would rather keep that